scripts/upgrade

```diff
@@ -184,9 +200,11 @@ ynh_store_file_checksum --file="$config"
 ynh_script_progression --message="Deploying..."
 
 pushd "$final_path"
+
 	php$phpversion artisan migrate:refresh --seed
 	php$phpversion artisan firefly-iii:upgrade-database
 	php$phpversion artisan passport:install
+	php artisan cache:clear
```

Comment:
```suggestion
	php$phpversion artisan cache:clear
```
you also need to declare `phpversion` variable in LOAD SETTINGS section -> `phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)`